finality-aleph/src/network/clique/protocols/v0/mod.rs

```diff
@@ -123,14 +138,45 @@ pub async fn incoming<SK: SecretKey, D: Data, S: Splittable>(
     }
 }
 
+pub async fn handle_authorization<SK: SecretKey>(
```

Comment:
Maybe just `authorize`? You are not handling an authorization here, just trying to get the connection authorized. But this is silly naming stuff, a complete nit, feel free to ignore. ^^"

Reply:
I choose `check_authorization`.